libs/tx-builder/src/utils/multicall.ts

```diff
@@ -145,6 +145,33 @@ export const handleEncodeCallArg = async ({
   return encodedData;
 };
 
+const handleMulticallFormActions = ({
```

Comment:
Curious to know what this is handling. Just want to make sure that it wouldn't overwrite anything.

Reply:
it reads the form state and if it holds encoded actions under `formState.tx`, appends them to the meta tx list to be sent in the tx proposal